lib/blockscore/util.rb

```diff
@@ -65,16 +65,16 @@ def to_constant(camel_cased_word)
     end
 
     def to_camelcase(str)
-      str.split('_').map(&:capitalize).join('')
+      str.split('_').map { |i| i.capitalize }.join('')
```

Comment:
Style/SymbolProc: Pass `&:capitalize` as an argument to `map` instead of a block.